piplicenses.py

```diff
@@ -190,7 +191,19 @@ def filter_string(item: str) -> str:
 
         return pkg_info
 
-    pkgs = importlib_metadata.distributions()
+    def get_python_sys_path(executable: str) -> list[str]:
+        script = "import sys; print(' '.join(filter(bool, sys.path)))"
+        output = subprocess.run(
+            [executable, "-c", script], capture_output=True
+        )
+        return output.stdout.decode().strip().split()
+
+    if args.python == sys.executable:
+        search_paths = sys.path
+    else:
+        search_paths = get_python_sys_path(args.python)
+
+    pkgs = importlib_metadata.distributions(path=search_paths)
```

Comment:
To resolve the points that Codecov points out, we need a test case that mocks the call to `importlib_metadata.distributions()`, for example. Any ideas on how to do this?

Of course, it is preferable to have test cases, but if it is too difficult, I will merge the PR as is.

Reply:
Interesting to that L203 is not covered, I would have thought that the L202 would not be covered since we don't test other python environments